### PR TITLE
ON-23283 # add isElementVisible package

### PR DESCRIPTION
**Pull request by divporter**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,4 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 
-- Validate Config function
+### Added
+
+- validateConfig function
+- isElementVisble function
+- definition service
+- referenceDataStore calss
```
Comment (simonmarklar) [inline CHANGELOG.md]:
```suggestion
- referenceDataStore class
```

File: packages/is-element-visible/index.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { FormTypes } from '@oneblink/types'
+import {
+  conditionalLogicService,
+  formElementsService,
+} from '@oneblink/sdk-core'
+
+export default function isShowing(
```
Comment (simonmarklar) [inline packages/is-element-visible/index.ts:7]: is it worth making a recursive version now or wait until we need one?

Comment (divporter) [reply packages/is-element-visible/index.ts:7]: like to see if the parent is visible and so on?

Comment (simonmarklar) [reply packages/is-element-visible/index.ts:7]: nevermind, `formElementsService.findFormElement` does the recursion.

**Pull request merged**